src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -39,18 +39,25 @@ const LastUsedContent: FC<Props & tType> = ({ articleId, locale, t }) => {
   return (
     <div {...classes('result')}>
       {article && (
-        <Link
-          {...classes('link')}
-          to={toEditArticle(article.id, article.articleType)}>
-          {article.title.title} ({t('article.lastUpdated')}{' '}
-          {article && formatDate(article.updated)})
-        </Link>
+        <Fragment>
+          <Link
+            {...classes('link')}
+            to={toEditArticle(article.id, article.articleType)}>
+            {article.title.title} ({t('article.lastUpdated')}{' '}
+            {article && formatDate(article.updated)})
+          </Link>
+          <EditMarkupLink
+            to={toEditMarkup(
+              articleId,
+              article.supportedLanguages.includes(locale)
+                ? locale
+                : article.supportedLanguages[0],
+            )}
+            title={t('editMarkup.linkTitle')}
+            inHeader={true}
+          />
+        </Fragment>
```

Comment:
Samme som over. Då treng du heller ikkje importere Fragment. Eller var det slik at det ikkje gjaldt for TS?
```suggestion
        </>
```